doc/visualizations.rst

```diff
@@ -20,24 +20,22 @@ like to only compute the predictions once and one should use `from_predictions`
 instead. In the following example, we plot a ROC curve for a fitted support
 vector machine:
 
-.. code-block:: python
+.. plot::
```

Comment:
This is nice. Would it be possible via some CSS to center the plot the same way we centered the figures?

Reply:
Updated PR with [`ccb737a` (#22579)](https://github.com/scikit-learn/scikit-learn/pull/22579/commits/ccb737a9d73d9997cce4a0c499ea91e4528ee5e0) to center images.